resources/views/auth/register.blade.php

```diff
@@ -61,11 +72,12 @@ class="mt-2"
             />
             <x-text-input
                 id="email"
-                class="mt-1 block w-full"
+                class="mt-1 block w-full {{ $githubEmail !== null ? 'text-slate-500' : '' }}"
```

Comment:
If we received the email from GitHub we mark it as readonly and with a lighter colour